upup/models/nodeup/networking/_cni_flannel/files/opt/cni/bin/flannel.asset

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Are these asset files generated in some code? If so there should be some documentation around that probably, but I can totally move it to code if the other asset files are generated in the same way